src/Pangraph.hs

```diff
@@ -135,41 +133,37 @@ vertexAttributes = vertexAttributes'
 edgeEndpoints :: Edge -> (Vertex, Vertex)
 edgeEndpoints = endpoints'
 
-{-| Returns the EdgeID if it has one. `Edge`s are given a new `VetexID` when they are passed and retrived from a `Pangraph`-}
+{-| Returns the EdgeID if it has one. 'Edge's are given a new 'EdgeID' when they are passed and retrived from a 'Pangraph'-}
 edgeID :: Edge -> Maybe EdgeID
 edgeID = edgeID'
 
--- | Returns a `VetexID`
+-- | Returns a 'VetexID'
 vertexID :: Vertex -> VertexID
 vertexID = vertexID'
 
 -- * Operations on Edge and Vertex
 
--- | Lookup Attributes by `Key`s
--- | Complexity: /O(n)/
+-- | Lookup Attributes by 'Key's Complexity: /O(n)/
 lookupVertexValues :: Key -> Vertex -> Maybe Value
 lookupVertexValues k v = lookup k (vertexAttributes v)
 
--- | Lookup Attributes by `Key`s
--- | Complexity: /O(n)/
+-- | Lookup Attributes by 'Key's Complexity: /O(n)/
 lookupEdgeValues :: Key -> Edge -> Maybe Value
 lookupEdgeValues k e = lookup k (edgeAttributes e)
 
 -- * Utility Operations
--- | Similar to `vertices` but returns an association list instead
+-- | Similar to 'vertices' but returns an association list instead
```

Comment:
I think it should remain in case the user would like to use a data structure that uses assoc lists as a constructor as it can save clock cycles.